src/main/java/seedu/address/model/book/BookStatus.java

```diff
@@ -119,6 +119,21 @@ public String getReturnDateString() {
         return STATUS_DATE_FORMAT.format(returnDate.get());
     }
 
+    /**
+     * Returns true if a book is overdue.
+     */
+    public boolean isOverdue() {
+        String returnDateString = getReturnDateString();
+        if (returnDate.equals("")) {
+            return false;
```

Comment:
Suggest using `if (isBorrowed())` to check whether book is borrowed instead. We should not use empty return date string to check for book borrowing status because this function might change in future, and not return empty string if the book is not borrowed.